Dnn.AdminExperience/ClientSide/SiteSettings.Web/src/components/siteLanguageSelector/index.jsx

```diff
@@ -65,35 +65,37 @@ class SiteLanguageSelector extends Component {
 
     onSiteChange(event) {
         let {state, props} = this;
-        props.dispatch(SearchActions.getCultureList(event.value, () => {
-            let culture = this.validateCultureCode();
-            if (state.cultureCode !== culture) {
-                this.setState({
-                    portalId: event.value,
-                    cultureCode: culture
-                });
-                this.triggerEvent("portalIdCultureCodeChanged",
-                    {
+        if(event.value !== state.portalId){
```

Comment:
Could we switch this around to reduce the nesting?

```js
if (event.value === state.portalId) {
    return;
}
```

Reply:
Updated